components/gallery/GalleryItem.vue

```diff
@@ -136,7 +146,9 @@ const { placeholder } = useTheme()
 const { nft, nftMetadata, nftImage, nftAnimation, nftMimeType } =
   useGalleryItem()
 const collection = computed(() => nft.value?.collection)
-const isMobile = ref(window.innerWidth < 768)
+
+const breakPointWidth = 930
```

Comment:
Why value change?

Reply:
to match the behavior of GallryIem page
line 220 ->
```scss
<style lang="scss" scoped>
@import '@/styles/abstracts/variables';
$break-point-width: 930px;
.title {
  font-size: 2.4375em;
}
...
```